src/qt/dc.cpp

```diff
@@ -745,17 +748,25 @@ bool wxQtDCImpl::DoBlit(wxCoord xdest, wxCoord ydest,
     if ( !qtSource )
         return false;
 
-    QImage qtSourceConverted = qtSource->toImage();
-    if ( !useMask )
-        qtSourceConverted = qtSourceConverted.convertToFormat( QImage::Format_RGB32 );
-
     // Change logical function
     wxRasterOperationMode savedMode = GetLogicalFunction();
     SetLogicalFunction( rop );
 
-    m_qtPainter->drawImage( QRect( xdest, ydest, width, height ),
-                           qtSourceConverted,
-                           QRect( xsrc, ysrc, width, height ) );
+    if ( useMask )
+    {
+        m_qtPainter->drawPixmap( QRect( xdest, ydest, width, height ),
```

Comment:
Minor: there is a form that will directly take the coordinates, without creating the in place `QRect`s.